src/tests/Interop/ExecInDefAppDom/ExecInDefAppDom.cs

```diff
@@ -81,10 +81,17 @@ public static int TestEntryPoint()
         result += TestExecuteInAppDomain(myPath, "FakeInjectedCode", "WrongReturnType", "None", COR_E_MISSINGMETHOD, 0);
         result += TestExecuteInAppDomain(myPath, "FakeInjectedCode", "Return0", "None", S_OK, 0);
         result += TestExecuteInAppDomain(myPath, "FakeInjectedCode", "Return1", "None", S_OK, 1);
-        result += TestExecuteInAppDomain(myPath, "FakeInjectedCode", "ThrowAnything", "None", COR_E_EXCEPTION, 0);
+        // Interpreter-FIXME: This requires EH interop which is not currently supported by the interpreter
```

Comment:
Is there anything to fix? I think this can just link to https://github.com/dotnet/runtime/issues/118965 without any fixmes or TODOs.